test/system/700-play.bats

```diff
@@ -1035,3 +1035,37 @@ spec:
     run_podman rm -f -t 0 -a
     run_podman rmi automount_test
 }
+
+@test "podman kube restore user namespace" {
+    skip_if_remote "userns=auto is set on the server"
+
+    if is_rootless; then
+        grep -E -q "^$(id -un):" /etc/subuid || skip "no IDs allocated for current user"
+    else
```

Comment:
subuid can be specified via name or uid so this seems to skip it in case it uses id
Anyhow how would we check rootless at all here? Most rootless tests will fail if you do not have subuid setup so I would drop this.

Reply:
I've copied the same test we have in "podman userns=auto in config file"

Reply:
I still think this line should be dropped

Reply:
dropped